swervelib/Mk4iSwerveModuleHelper.java

```diff
@@ -355,10 +585,94 @@ public static SwerveModule createFalcon500Neo(
             int steerEncoderPort,
             double steerOffset
     ) {
+        // return new SwerveModuleFactory<>(
+        //         gearRatio.getConfiguration(),
+        //         getFalcon500DriveFactory(configuration),
+        //         getNeoHeliumSteerFactory(configuration)
+        // ).create(
+        //         driveMotorPort,
+        //         new SteerConfiguration<>(
+        //                 steerMotorPort,
+        //                 new CanandCoderAbsoluteConfiguration(steerEncoderPort)
+        //         )
+        // );
+        if (configuration.getSteerEncoderType() == AbsoluteEncoderType.HELIUM_CANCODER) {
+        
+                return new SwerveModuleFactory<>(
+                        gearRatio.getConfiguration(),
+                        getFalcon500DriveFactory(configuration),
+                        getNeoHeliumSteerFactory(configuration)
+                ).create(
+                        driveMotorPort,
+                        new SteerConfiguration<>(
+                                steerMotorPort,
+                                new CanandCoderAbsoluteConfiguration(steerEncoderPort)
+                        )
+                );  
+        }
         return new SwerveModuleFactory<>(
                 gearRatio.getConfiguration(),
                 getFalcon500DriveFactory(configuration),
-                getNeoCanCoderSteerFactory(configuration)// ADD HELIUM LOGIC HERE
+                getNeoCanCoderSteerFactory(configuration) 
+        ).create(
+                driveMotorPort,
+                new SteerConfiguration<>(
+                        steerMotorPort,
+                        new CanCoderAbsoluteConfiguration(steerEncoderPort, steerOffset)
+                )
+        );
+
+        
+    }
+
+    /**
+     * Creates a Mk4i swerve module that uses a Kraken X60 for driving and a NEO for steering.
+     *
+     * @param configuration    Module configuration parameters to use.
+     * @param gearRatio        The gearing configuration the module is in.
+     * @param driveMotorPort   The CAN ID of the drive Kraken X60.
+     * @param steerMotorPort   The CAN ID of the steer NEO.
+     * @param steerEncoderPort The CAN ID of the steer CANCoder.
+     * @param steerOffset      The offset of the CANCoder in radians.
+     * @return The configured swerve module.
+     */
+    public static SwerveModule createKrakenX60Neo(
+            Mk4ModuleConfiguration configuration,
+            GearRatio gearRatio,
+            int driveMotorPort,
+            int steerMotorPort,
+            int steerEncoderPort,
+            double steerOffset
+    ) {
+        // return new SwerveModuleFactory<>(
+        //         gearRatio.getConfiguration(),
+        //         getKrakenX60DriveFactory(configuration),
```

Comment:
don't leave commented out stuff like this. we need it or we don't need it. if we don't need it, delete it